```cmake
2021-08-23T08:18:17.9025201Z CMake Error at ports/autodock-vina/portfile.cmake:15 (vcpkg_cmake_configure):
2021-08-23T08:18:17.9030098Z   Unknown CMake command "vcpkg_cmake_configure".
2021-08-23T08:18:17.9035742Z Call Stack (most recent call first):
2021-08-23T08:18:17.9040474Z   scripts/ports.cmake:140 (include)
```

--- a/ports/autodock-vina/01-windows.patch
+++ b/ports/autodock-vina/01-windows.patch
@@ -1,0 +1,12 @@
+diff --git a/src/lib/forcefield.h b/src/lib/forcefield.h
+index 81bd4c1..44761e2 100644
+--- a/src/lib/forcefield.h
++++ b/src/lib/forcefield.h
+@@ -24,6 +24,7 @@
+ #define VINA_FORCEFIELD_H
+ 
+ 
++#include <sstream>
+ #include <iostream>
+ #include <fstream>
+ #include <string>

--- a/ports/autodock-vina/CMakeLists.txt
+++ b/ports/autodock-vina/CMakeLists.txt
@@ -1,0 +1,100 @@
+cmake_minimum_required(VERSION 3.11)
+cmake_policy(VERSION 3.11)
+
+project(autodock-vina)
+
+set(GIT_VERSION v1.2.2)
+set(CMAKE_CXX_STANDARD 11)
+
+find_package(Boost REQUIRED COMPONENTS 
+    filesystem
+    log
+    math_tr1
+    program_options
+    random
+    serialization
+    thread
+    timer
+)
+
+file(GLOB SRC_AUTODOCK_VINA_LIBS
+     "src/lib/*.cpp"
+)
+
+file(GLOB SRC_AUTODOCK_VINA_SPLIT
+     "src/split/*.cpp"
+)
+
+file(GLOB HEADERS
+     "src/lib/*.h"
+)
+
+add_library(vina ${SRC_AUTODOCK_VINA_LIBS})
+add_library(vina_split ${SRC_AUTODOCK_VINA_SPLIT})
+
+target_include_directories(vina 
+     PUBLIC 
+          $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src/lib/> 
+          $<INSTALL_INTERFACE:include>
+     PRIVATE 
+          $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>
+)
+
+target_include_directories(vina_split 
+     PUBLIC 
+          $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src/lib/> 
+          $<INSTALL_INTERFACE:include>
+     PRIVATE 
+          $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>         
+)
+
+target_compile_definitions(vina 
+     PUBLIC 
+          -DVERSION=\"${GIT_VERSION}\"
+)
+
+target_compile_definitions(vina_split 
+     PUBLIC 
+          -DVERSION=\"${GIT_VERSION}\"
+)
+
+target_link_libraries(vina 
+     PRIVATE 
+        Boost::boost
+        Boost::filesystem
+        Boost::log
+        Boost::math_tr1
+        Boost::program_options
+        Boost::random
+        Boost::serialization
+        Boost::thread
+        Boost::timer
+)
+
+target_link_libraries(vina_split 
+     PRIVATE 
+        Boost::boost
+        Boost::filesystem
+        Boost::log
+        Boost::math_tr1
+        Boost::program_options
+        Boost::random
+        Boost::serialization
+        Boost::thread
+        Boost::timer
+)
+
+install(TARGETS vina EXPORT autodock-vina-config
+            RUNTIME DESTINATION bin
+            ARCHIVE DESTINATION lib
+            LIBRARY DESTINATION lib
+)
+
+install(TARGETS vina_split EXPORT autodock-vina-config
+            RUNTIME DESTINATION bin
+            ARCHIVE DESTINATION lib
+            LIBRARY DESTINATION lib
+)
+
+install(FILES ${HEADERS} DESTINATION include/autodock-vina)
+install(EXPORT autodock-vina-config NAMESPACE autodock-vina::autodock-vina:: DESTINATION share/autodock-vina)

--- a/ports/autodock-vina/portfile.cmake
+++ b/ports/autodock-vina/portfile.cmake
@@ -1,0 +1,32 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO ccsb-scripps/AutoDock-Vina
+    REF v1.2.2
+    SHA512 a6117137292b9b77afd3dff4a5ca31c65c9e85edd0b2351795fb2cccc06eca5b865b7dc054bb2859d0cf32abb6f6a72aaf62647e6d53709be879cc12a0336ba8
+    HEAD_REF develop
+    PATCHES 
+        01-windows.patch
+)
+
+file(COPY "${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt" DESTINATION "${SOURCE_PATH}")
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+)
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup()
+
+vcpkg_copy_pdbs()
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
+
+file(
+    INSTALL "${SOURCE_PATH}/LICENSE"
+    DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}"
+    RENAME copyright
+)

--- a/ports/autodock-vina/vcpkg.json
+++ b/ports/autodock-vina/vcpkg.json
@@ -1,0 +1,31 @@
+{
+  "name": "autodock-vina",
+  "version-semver": "1.2.2",
+  "description": "AutoDock Vina is one of the fastest and most widely used open-source docking engines.",
+  "homepage": "http://vina.scripps.edu/",
+  "dependencies": [
+    "boost-algorithm",
+    "boost-array",
+    "boost-filesystem",
+    "boost-lexical-cast",
+    "boost-log",
+    "boost-math",
+    "boost-optional",
+    "boost-program-options",
+    "boost-ptr-container",
+    "boost-random",
+    "boost-serialization",
+    "boost-static-assert",
+    "boost-thread",
+    "boost-timer",
+    "boost-utility",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
+}

--- a/versions/a-/autodock-vina.json
+++ b/versions/a-/autodock-vina.json
@@ -1,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "393884054347c19ff78db882674ce17dd51476e0",
+      "version-semver": "1.2.2",
+      "port-version": 0
+    }
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -228,6 +228,10 @@
       "baseline": "20.8.1",
       "port-version": 0
     },
+    "autodock-vina": {
+      "baseline": "1.2.2",
+      "port-version": 0
+    },
     "avcpp": {
       "baseline": "2021-06-14",
       "port-version": 0

